### PR TITLE
Onion Architecture

### PR DESCRIPTION
**Pull request by Rigorich**

### CODE REVIEW

File: Buildlease/Contracts/Views/CaregoryView.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Contracts.Views
+{
+    public class CaregoryView
```
Comment (rosteeslove) [inline Buildlease/Contracts/Views/CaregoryView.cs]: typo

**Review by Rigorich — COMMENTED** [review]
+rep (to rasteeslove)

**Comment by aperaverzeu** [comment]
Add DTO folder

**Review by aperaverzeu — APPROVED** [review]
Good, we will do more functionality in a new and fully develped api

**Review by Stanislav-3 — APPROVED** [review]

**Review by rosteeslove — APPROVED** [review]
Great job!

**Pull request merged**